Replicated/Replicated.cpp

```diff
@@ -20,11 +20,18 @@ void Replicated::AddBlockToLevel(GameObject *g, GameWorld& world, PrimitiveGameO
     world.AddGameObject(g, currentPrimitive->shouldNetwork);
     auto volume = new OBBVolume(currentPrimitive->colliderExtents * 0.5f);
     g->SetBoundingVolume((CollisionVolume*)volume);
+    std::cout << currentPrimitive->colliderExtents << "\n";
+    Vector3 tempFix = (currentPrimitive->rotation).Quaternion::ToEuler();
 
+    tempFix.x = -tempFix.x;
```

Comment:
I've changed this to a vector3 multiplication rather than setting each value manually.